packages/react/src/field/Field.tsx

```diff
@@ -103,9 +106,12 @@ export const Field = forwardRef<HTMLDivElement, FieldProps>(
         </FieldContext.Provider>
 
         {error && (
-          <Box asChild color="fg.error" fontSize="sm">
+          <Flex color="fg.error" flexDirection="row" fontSize="sm" gap="4">
+            <Box asChild {...styles.icon()}>
```

Comment:
let's do align start on the flex and use margin-top on the icon to align it vertically - that way we can handle multi-line error messages

also we need a flex none on the icon - so it doesn't shrink when there's a lot of text

```suggestion
          <Flex alignItems="start" color="fg.error" flexDirection="row" fontSize="sm" gap="4">
            <Box asChild flex="none" mt="2" {...styles.icon()}>
```

<img width="307" alt="Screenshot 2024-10-21 at 4 58 30 PM" src="https://github.com/user-attachments/assets/4b91f2f8-8e98-4d8e-8a20-bd0c49b1a7b0">

Reply:
Okay vai